samples/authorization/csm_authorization_proxy_server_v230.yaml

```diff
@@ -99,16 +110,16 @@ spec:
               - secret-provider-class-1
               - secret-provider-class-2
             # conjur:
-              # - name: secret-provider-class-3
-              #   paths:
-                  # - usernamePath: secrets/username1
-                  #   passwordPath: secrets/password1
-                  # - usernamePath: secrets/username2
-                  #   passwordPath: secrets/password2
-              # - name: secret-provider-class-4
-                # paths:
-                  # - usernamePath: secrets/username3
-                  #   passwordPath: secrets/password3
+            # - name: secret-provider-class-3
+            #   paths:
+            # - usernamePath: secrets/username1
+            #   passwordPath: secrets/password1
+            # - usernamePath: secrets/username2
+            #   passwordPath: secrets/password2
+            # - name: secret-provider-class-4
+            # paths:
```

Comment:
Can we revert this change? Indentation/sample is not right and will be confusing.

Reply:
I think I got the lint error here because of the indentation and the check would not pass here